Toast/ToastViews/DismissibleTitleMessageToastView.cs

```diff
@@ -57,6 +57,8 @@ protected override void ConstrainChildren()
 
         private void DismissButton_TouchUpInside(object sender, EventArgs e)
         {
+            Toast.DismissCallBack?.Invoke();
```

Comment:
I think this has to be called in `BaseToastView.cs`, somewhere around here: https://github.com/andrius-k/Toast/blob/master/Toast/ToastViews/BaseToastView.cs#L184

Otherwise the callback will be called only for this specific view.